.github/workflows/check.yaml

```diff
@@ -20,17 +20,62 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  lint-unit:
-    uses: canonical/bootstack-actions/.github/workflows/lint-unit.yaml@v2
+  lint:
+    name: Lint
     strategy:
       fail-fast: false
       matrix:
         python-version: ["3.8", "3.10"]
-    with:
-      python-version: ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
```

Comment:
Ideally this should be 22.04

Reply:
Good catch; this will need to be updated in the automation repo at https://github.com/canonical/solutions-engineering-automation/blob/c82b52005500591c192ffff5af391bb99eb657b4/terraform-plans/templates/github/charm_check.yaml.tftpl#L29

Reply:
Added in https://github.com/canonical/solutions-engineering-automation/pull/86